subsys/bluetooth/host/gatt.c

```diff
@@ -4587,13 +4592,41 @@ int bt_gatt_resubscribe(uint8_t id, const bt_addr_le_t *peer,
 	return 0;
 }
 
+static bool sub_has_subscription(struct gatt_sub *sub,
+	const struct bt_gatt_subscribe_params *params)
+{
+	struct bt_gatt_subscribe_params *other_params;
+
+	SYS_SLIST_FOR_EACH_CONTAINER(&sub->list, other_params, node) {
+		if (params == other_params) {
+			return true;
+		}
+	}
+
+	return false;
+}
+
+static bool sub_has_other_subscriptions_to_same_attribute(struct gatt_sub *sub,
```

Comment:
You really seem to like long symbol names 😉 

How about just `has_other_sub()`?